driver-moab/main/seq_rest_mod.F90

```diff
@@ -666,7 +667,9 @@ subroutine seq_rest_write(EClock_d, seq_SyncClock, infodata, &
     type(mct_aVect)        , intent(inout) :: fractions_wx(:)   ! Fractions on wav grid/decomp
     type(mct_aVect)        , intent(inout) :: fractions_zx(:)   ! Fractions on iac grid/decomp
     character(len=*)       , intent(in)    :: tag
-    character(len=CL)      , intent(out)   :: rest_file         ! Restart filename
+    character(len=CL)      , intent(out)   :: rest_file_org         ! Restart filename
+
```

Comment:
I thought the idea was to write to a single restart file from MOAB (not MCT) so that we can test against MCT driver separately.